src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -200,13 +239,13 @@ public Task getTask(String id,
         );
     }
 
-    private Map<String, CamundaVariable> performGetVariablesAction(String id) {
+    public Map<String, CamundaVariable> performGetVariablesAction(String id) {
         Map<String, CamundaVariable> variables;
         try {
             variables = camundaServiceApi.getVariables(authTokenGenerator.generate(), id);
         } catch (FeignException ex) {
             throw new ResourceNotFoundException(String.format(
-                "There was a problem fetching the task with id: %s",
+                "There was a problem updating the task with id: %s. The task could not be found.",
```

Comment:
This will probably break all tests when we retrieve the variables we aren't really updating the task so i think fetching or retrieving was correct. 
Otherwise you will get a 404 saying 'There was a problem updating the task '

Reply:
yes, you're right.  I changed it to match I think some functional test expectations. I am not sure now. In any case, it makes sense. I will change it.

Reply:
done